README.md

```diff
@@ -454,63 +447,53 @@ matches-ignore:
 
 ## Environment Variables
 
-Some configurations on `ggshield` can be done through environment variables.
+ggshield can also be configured using environment variables.
 
-Environment variables will override settings set on your config file but will be overridden by command line options.
+Environment variables overrides settings set on your configuration file but are overridden by command line options.
 
-At startup, `ggshield` will attempt to load environment variables from different environment files in the following order:
+At startup, ggshield attempts to load environment variables from different environment files in the following order:
 
 - path pointed to by the environment variable `GITGUARDIAN_DOTENV_PATH`
 - `.env` at your current work directory.
 - `.env` at the root of the current git directory
 
 Only one file will be loaded of the three.
 
-Reference of current Environment Variables that affect `ggshield`:
+Reference of current environment variables supported by ggshield:
 
-```yaml
-GITGUARDIAN_API_KEY: [Required] API Key for the GitGuardian API.
+- `GITGUARDIAN_API_KEY`: API Key for the GitGuardian API. Use this if you don't want to use the `ggshield auth` commands.
 
-GITGUARDIAN_URL: Custom URL of the GitGuardian dashboard. The API URL will be inferred from it.
+- `GITGUARDIAN_INSTANCE`: Custom URL of the GitGuardian dashboard. The API URL will be inferred from it.
```

Comment:
These modifications of env variables need to be modified in public doc as well. Can you open a MR there please?